### PR TITLE
HumanCompatibleAI -> AlignmentResearch (v1)

### PR DESCRIPTION
**Pull request by ed1d1a8d**
Updates README and .gitmodules to reflect migration to AlignmentResearch github org.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	url = git@github.com:HumanCompatibleAI/gogui.git
```
Comment (tomtseng) [inline .gitmodules:3]:
```suggestion
	url = git@github.com:AlignmentResearch/gogui.git
```

**Review by tomtseng — COMMENTED** [review]
there's a git clone in each of  `/engines/elf/Dockerfile`, `/engines/leela/Dockerfile`, and `compose/transfer/twogtp/Dockerfile` that will need to be changed as well. Grepping for "HumanCompatibleAI" should find everything

File: .gitmodules (hunk 1 above)
Comment (ed1d1a8d) [reply .gitmodules:3]: Will wait on these repos to be migrated and then ammend this.

**Comment by ed1d1a8d** [comment]
Closing because README has since been updated to incorporate many of the changes in this PR.

**Comment by AdamGleave** [comment]
Reopening now I've moved gogui

**Comment by ed1d1a8d** [comment]
Can't actually review cause I am the original PR author, but LGTM.
I checked and the only references to "HumanCompatibleAI" (non-case-sensitive) are for docker hub.
I made a new issue to track the docker migration when we eventually get to it: https://github.com/AlignmentResearch/go_attack/issues/104

**Review by AdamGleave — APPROVED** [review]
Self-LGTM as Tony LGTM'd in comment.

**Pull request merged**